tests/network/hotplug.go

```diff
@@ -47,49 +48,35 @@ import (
 	"kubevirt.io/kubevirt/tests/util"
 )
 
-var _ = SIGDescribe("nic-hotplug", decorators.InPlaceHotplugNICs, func() {
+type hotplugMethod string
+
+const (
+	migrateBased hotplugMethod = "migrationBased"
```

Comment:
nit: Please consider renaming it to `migrationBased`.

Reply:
done.